src/AssertWrapper.sol

```diff
@@ -59,4 +59,8 @@ abstract contract AssertWrapper is AssertHelper {
     function t(bool a, string memory message) internal {
         assertWithMsg(a, message);
     }
+
+    function f(bool a, string memory message) internal {
+        assertWithMsg(!a, message);
+    }
```

Comment:
I'm not sure if this is actually necessary, since the user should be negating their boolean and using the t() function, I think adding the `f(...)` function may be encouraging bad design.

I believe it's better design to always try to pass a true to an assertion, then if it doesn't happen it breaks the invariant. For example, when I do a `gt(a, b)`, it's like I'm writing `a > b`. When `a <= b` happens, it breaks the invariant. 

However, if I do `f(...)`, it's like I'm doing `a == false`, which feels like bad design. I think it should just be `!a`, and thus the `f(...)` function is not necessary.

Reply:
Decided not to add this